client/src/main/java/com/arcbees/beeshop/client/application/widget/sidepanel/cart/ShoppingBagPresenter.java

```diff
@@ -61,6 +62,11 @@ public void onShoppingBagChanged(ShoppingBagChangedEvent event) {
     }
 
     @Override
+    public void checkout() {
+        CheckoutContinueEvent.fire(this);
+    }
+
+    @Override
     protected void onBind() {
         addRegisteredHandler(ShoppingBagChangedEvent.TYPE, this);
```

Comment:
Nope, it's the only thing that adds items to the shopping cart, and the cart is not always visible.